build/webpack.config.babel.js

```diff
@@ -8,29 +8,36 @@ const babelConfig = require('../.babelrc.js')
 const getVersion = require('git-repo-version')
 
 const createConfig = (env = {}) => {
-  if (env.prod === undefined) {
-    throw Error('You need to specify a mode!')
+  const { config_name: appConfigName, prod } = env
+  const validConfigNames = ['integreat', 'integreat-debug', 'malte']
+
+  if (!appConfigName) {
+    throw new Error('You need to specify a config name!')
+  } else if (!validConfigNames.includes(appConfigName)) {
+    throw new Error(`Invalid config name! Allowed configs: ${validConfigNames}`)
   }
 
-  const isDebug = env.prod === 'false'
-  const appConfigName = env.config_name || 'integreat'
-  const appConfig = require(`./${appConfigName}-config`)
+  // If prod is not specified manually, default to true for 'integreat' and 'malte' configs, false for 'integreat-debug'
+  const production = prod || ['integreat', 'malte'].includes(appConfigName)
```

Comment:
when looking at the package.json prod is not defined in Debug mode. Shouldn't it be the other way around then?